.github/actions/setup-builder/action.yaml

```
@@ -0,0 +1,38 @@
+name: Prepare Rust Builder
+description: 'Prepare Rust Build Environment'
+inputs:
+  rust-version:
+    description: 'version of rust to install (e.g. stable)'
+    required: true
+    default: 'stable'
+runs:
+  using: "composite"
+  steps:
+    - name: Install Build Dependencies
+      shell: bash
+      run: |
+        apt-get update
+        apt-get install -y protobuf-compiler
```

Looks like we need to `sudo` these installs:

```suggestion
        sudo apt-get update
        sudo apt-get install -y protobuf-compiler
```

https://stackoverflow.com/questions/57982945/how-to-apt-get-install-in-a-github-actions-workflow